mkdocs.yml

```diff
@@ -57,6 +57,7 @@ nav:
       - NER model configuration: samples/python/ner_model_configuration.ipynb
       - Presidio Structured Basic Usage: samples/python/example_structured.ipynb
       - Using an allow list with image redaction: samples/python/image_redaction_allow_list_approach.ipynb
+      - LiteLLM (OpenAI Proxy): samples/docker/litellm.md
```

Comment:
Please add to the index.md on the samples folder. These are only the most fundamental notebooks for Presidio.

Reply:
resolved